src/error.rs

```diff
@@ -47,7 +47,7 @@ impl StdError for Error {
     fn cause(&self) -> Option<&StdError> {
         match *self {
             Error::Utf8(ref err) => Some(err),
-            Error::Xml(ref err) => Some(err),
+            // Error::Xml(ref err) => Some(err.cause(),
```

Comment:
should this still be commented out?

Reply:
No, but I can't for the life of me figure out how to convert a quick-xml error into a StdError…

Reply:
fixed in https://github.com/rust-syndication/rss/commit/c23ee3cc8e2228c4e1c1e84a501aacf95f7d3502